Casks/umts-dev.rb

```diff
@@ -12,9 +12,20 @@
 
   stage_only true
 
+  # version managers
   depends_on formula: 'nodenv'
   depends_on formula: 'rbenv'
 
+  # ruby dependencies
+  depends_on formula: 'libyaml'
+  depends_on formula: 'openssl'
+  depends_on formula: 'readline'
+
+  # rails dependencies
+  depends_on formula: 'mysql'
+  depends_on formula: 'imagemagick'
+  depends_on formula: 'vips'
```

Comment:
We talked at the standup, but I think these sort of things should go in the individual repositories that rely on them. Yes, we use mysql/mariadb on most of our apps, but I think sticking to sqlite while working on tutorials is fine. It doesn't look like any of the tutorials use Active Storage at all, and the two apps we have that do only use `imagemagick`.